File: beginners/uri/2-problemset-condicionais/1035-teste-selecao-1/teste_selecao1.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B, C, D, somaCD, somaAB;

    cin >> A >> B >> C >> D;

    somaCD = C + D;
    somaAB = A + B;

    if ((B > C && D > A) && (somaCD > somaAB && C > 0 && D > 0) && (A % 2 == 0))
        cout << "Valores aceitos" << endl;
    else
        cout << "Valores nao aceitos" << endl;

    return 0;
}

File: beginners/2-problemset-condicionais/Python/1035-teste-selecao-1/teste_selecao1.py
a, b, c, d = input().split()
a = int(a)
b = int(b)
c = int(c)
d = int(d)

somacd = c + d
somaab = a + b

if b > c and d > a and somacd > somaab and c > 0 and d > 0 and a % 2 == 0:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')
